.coderabbit.yaml

```
@@ -0,0 +1,343 @@
+      - unconvert
+      - gocyclo
+      - dupl
+      - goconst
+      - gosec
+      - staticcheck
+
+# Review settings
+reviews:
+  # Enable auto-review on all PRs
+  auto_review_enabled: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)